tests/test_send_recv_two_workers.py

```diff
@@ -251,9 +251,8 @@ def test_send_recv_cu(cuda_obj_generator):
 
 
 def total_nvlink_transfer():
-    pynvml.nvmlShutdown()
     pynvml.nvmlInit()
```

Comment:
This is certainly an oddity. Makes sense to drop. Though it is curious why it was there.

Reply:
This was just poor coding because I had previously initialized in another function